config/overlays/certs/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+namespace: olmv1-system
+
+resources:
+- ../../overlays/tls
```

Comment:
Do I understand correctly that `config/overlays/certs` only exists for E2Es? If yes, would it make sense to put this into `config/overlays/e2e`?

Reply:
The `e2e` has coverage that's not needed for the ext-dev-e2e, but the ext-dev-e2e needs certs as well.

Reply:
It looks like I commented about the same thing twice in different places. Let's continue here: https://github.com/operator-framework/operator-controller/pull/960/files#r1648592694

Closing this thread.